SI/Sem.07/Pract.07/References/Task02.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+const char UPPER_BORDER = 'Z';
+const char LOWER_BORDER = 'z';
+const char UPPER_BORDER_TO_UP = 'A';
+const char LOWER_BORDER_TO_SMALL = 'a';
+void toUpper(char& a)
+{
+	if (a >= LOWER_BORDER_TO_SMALL && a <= LOWER_BORDER)
+	{
+		a -= 32;
+	}
+	else
+	{
+		std::cout << false;
+	}
+}
+void toLower(char& b)
+{
+	if (b >= UPPER_BORDER_TO_UP && b <= UPPER_BORDER)
+	{
+		b += 32;
+	}
+	else
+	{
+		std::cout << false;
+	}
+}
+int main()
+{
+	char symbol;
+	std::cin >> symbol;
+	if (symbol >= LOWER_BORDER_TO_SMALL && symbol <=LOWER_BORDER)
+	{
+		toUpper(symbol);
+	}
```

Comment:
You are doing the same check in the toUpperFunction
The same for the code below with the toLower
Upload only the two function - toUpper and toLower!